tutorials/cyber/flow_classification/flow_classification_rapids.ipynb

```diff
@@ -0,0 +1,1322 @@
+    "print(\"==> pdf shape: \",pdf.shape)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can look at what this new aggregated data looks like, and get a better sense of the columns and their data types. Let's do this the way we're familiar with, using Pandas."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pdf.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pdf.dtypes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "That's Pandas, and we could certinaly continue the analysis there if we wanted. But what about  [cuDF](https://github.com/rapidsai/cudf)? Let's pivot to that for the majority of this tutorial.\n",
+    "\n",
+    "One thing cuDF neeeds is for us to specify the data types. We'll write a function to make this easier. One thing that is coming very soon to cuDF is integration of string support. Let's try  it out here before it's officially released."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_dtypes(fn, delim, floats, strings):\n",
+    "    with open(fn, errors='replace') as fp:\n",
+    "        header = fp.readline().strip()\n",
+    "    \n",
+    "    types = []\n",
+    "    for col in header.split(delim):\n",
+    "        if 'date' in col:\n",
+    "            types.append((col, 'date'))\n",
+    "        elif col in floats:\n",
+    "            types.append((col, 'float64'))\n",
+    "        elif col in strings:\n",
+    "            types.append((col, 'str'))\n",
+    "        else:\n",
+    "            types.append((col, 'int64'))\n",
+    "\n",
+    "    return OrderedDict(types)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dtypes_data_processed = get_dtypes(DATA_SOURCE, '\\t', floats=['ts','duration'],\n",
+    "                             strings=['uid','id.orig_h','id.resp_h','proto','service',\n",
+    "                                      'conn_state','local_orig','local_resp',\n",
+    "                                      'history','tunnel_parents','orig_l2_addr',\n",
+    "                                      'resp_l2_addr'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "raw_cdf = cd.io.csv.read_csv(DATA_SOURCE, delimiter='\\t', names=list(dtypes_data_processed), \n",
+    "                                       dtype=list(dtypes_data_processed.values()), skiprows=1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Strings are treated just like any other column (at least as far ast the user is concerned)."
```

Comment:
typo in this line (... as far as the user ...)
I was also wondering if we could edit the sentence to provide some information to the user with respect to the handling of string data

Reply:
@Salonijain27 What type of information would you want the user provided with here? How much in the weeds?

Reply:
Not a lot of information, if possible just an overview.

Reply:
@Salonijain27 I edited this line and linked to the "String Methods" in the "10 Minutes to cuDF" documentation.